### PR TITLE
Next gen

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

**Comment by joliveros** [comment]
LGTM @hortega!

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,6 @@ install:
 services: mongodb
 env:
   global:
-    - TWILIO_ACCOUNT_SID=Your-Account-SID
+    - TWILIO_ACCOUNT_SID=ACYour-Account-SID
```
Comment (acamino) [inline .travis.yml]: Not sure about this change. Looks like you introduced a typo. :sweat_smile:

File: package.json [hunk 2]
```diff
@@ -6,9 +6,10 @@
     "start": "node ./bin/www",
     "test": "export NODE_ENV=test && mocha spec"
   },
-  "engines" : { "node" : ">=4.1.0 <5.5.0" },
+  "engines": {
+    "node": ">=4.1.0 <5.5.0"
+  },
   "dependencies": {
-    "bluebird": "^3.1.1",
```
Comment (acamino) [inline package.json]: Thanks for removing this dependency and using the built-in promises. Removing external unnecessary dependencies is always a good thing. 😉

File: package.json [hunk 3]
```diff
@@ -25,7 +26,7 @@
     "passport-local-mongoose": "^3.1.0",
     "passport-stub": "^1.0.1",
     "serve-favicon": "~2.3.0",
-    "twilio": "^2.8.0"
+    "twilio": "^3.0.0-rc.13"
```
Comment (acamino) [inline package.json]: Would you mind using the strict version of this dependency?

`^3.0.0-rc.13` actually means `>=^3.0.0-rc.13 and <=4.0.0` -- I don't think we'd like to introduce such level of compatibility :cold_sweat:. If you want to talk about it please ping me.

File: package.json (hunk 2 above)
Comment (hortega) [inline package.json]: No prob! I totally agree

**Comment by hortega** [comment]
Thanks for the review @acamino !

File: .travis.yml (hunk 1 above)
Comment (acamino) [inline .travis.yml]: Thanks for the clarification.

**Comment by acamino** [comment]
Thanks @hortega! LTGM

**Pull request merged**